### PR TITLE
restaurants

### PR DESCRIPTION
**Pull request by kevinly77**
Added restaurants and finished wookie quotes

### CODE REVIEW

File: src/data/media/starwars.rs [hunk 1]
```diff
@@ -1,9 +1,8 @@
 use rand::Rng;
+use std::collections::HashMap;
 use wasm_bindgen::prelude::*;
-//Ask about making call_sign
-//Ask about how to deal with quotes and alternative character spellings
 
-//TODO left off at wookie words
+//TODO Ask again how to deal with quotes
```
Comment (clearfeld) [inline src/data/media/starwars.rs:4]: What questions do you have in regards to quotes?

File: src/data/media/starwars.rs [hunk 2]
```diff
@@ -52,6 +51,11 @@ pub fn vehicle() -> String {
 	let mut rng = rand::thread_rng();
 	VEHICLES[rng.gen_range(0..VEHICLES_LEN)].to_string()
 }
+
```
Comment (clearfeld) [inline src/data/media/starwars.rs:54]: The following functions should have #[wasm_bindgen] not just the first one.

File: src/data/misc/restaurant.rs [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+use wasm_bindgen::prelude::*;
+// ask about ### and ???
+// TODO Ask about generating a restuarant object?
+
+#[wasm_bindgen]
+pub fn name_prefix() -> String {
+	let mut rng = rand::thread_rng();
+	NAME_PREFIX[rng.gen_range(0..NAME_PREFIX_LEN)].to_string()
+}
+
```
Comment (clearfeld) [inline src/data/misc/restaurant.rs:11]: The following functions should have #[wasm_bindgen] not just the first one.

File: src/data/misc/restaurant.rs [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+	let mut rng = rand::thread_rng();
+	RESTAURANT_DESCRIPTION[rng.gen_range(0..RESTAURANT_DESCRIPTION_LEN)].to_string()
+}
+
+pub fn restaurant_review() -> String {
+	let mut rng = rand::thread_rng();
+	RESTAURANT_REVIEW[rng.gen_range(0..RESTAURANT_REVIEW_LEN)].to_string()
+}
+
+pub fn generate_full_restaurant() -> String {
```
Comment (clearfeld) [inline src/data/misc/restaurant.rs:44]: This is fine, but I do think we should discuss how to handle custom formatted data. So there's a standard across the library.

**Review by clearfeld — CHANGES_REQUESTED** [review]

File: src/data/media/starwars.rs (hunk 2 above)
Comment (kevinly77) [reply src/data/media/starwars.rs:54]: Sorry, I must of forgotten in the copy paste frenzy. I will fix it

File: src/data/misc/restaurant.rs (hunk 4 above)
Comment (kevinly77) [reply src/data/misc/restaurant.rs:44]: That was actually one of my questions I was going to ask. If we should return and object instead or something. Lets discuss this whenever you have some time in a call

File: src/data/media/starwars.rs (hunk 1 above)
Comment (kevinly77) [reply src/data/media/starwars.rs:4]: Should I just make each person with quotes their own array with corresponding method?
Is there a way to make it a Hashmap or some type of data structure? (unsure how to do this in Rust)
Lets discuss it in a quick call after the weekend

**Review by clearfeld — APPROVED** [review]

**Pull request merged**